discovery-utils/src/discovery/v0.rs

```diff
@@ -0,0 +1,408 @@
+#[derive(Clone, PartialEq, ::prost::Message)]
```

Comment:
this is generated, right?  i wish we could add a persisted comment that said it was generated and what generated it.

Reply:
seems like other folks wanted the same support from `prost` the rust protobuf library; however, there have been [no updates on the pr](https://github.com/danburkert/prost/pull/300) since April 2020.

Reply:
Added a comment that it is auto-generated where the module is declared in  0f7232d5d7